lib/puma/dsl.rb

```diff
@@ -664,6 +664,14 @@ def shutdown_debug(val=true)
       @options[:shutdown_debug] = val
     end
 
+    # Controls whether to inject small delay before
+    # accepting new socket
+    # if we are already processing requests
+    # this allows less busy workers to pick before us
+    def wait_for_less_busy_worker(val=true)
```

Comment:
Should we mention here this only does something on MRI?

Reply:
Good idea.